src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
Consider storing the instance of ColorSupplier as a class-level field instead of creating a new instance every time the method is called.

Reply:
not fixed